util/helpers.py

```diff
@@ -60,6 +93,20 @@ def parse_arguments(argv):
                         required=True,
                         help="Path of the output project directory")
 
+    parser.add_argument("-b",
+                        "--save_bitstream",
+                        dest="save_bitstream",
+                        type=bool,
+                        required=False,
+                        help="Save bitstream into project database")
+
+    parser.add_argument("-f",
+                        "--save_binary",
+                        dest="save_binary",
+                        type=bool,
+                        required=False,
+                        help="Save binary into project database")
+
```

Comment:
In my view, I would always save both or none of them. Or do you have a specific scenario in mind where only one is useful?

Reply:
When we do not use an FPGA in the future but still have the capabilities of changing the binary on the discrete chip, this could be useful.

Reply:
Ah yes, of course. You're right!